services/src/main/java/com/linkedin/kmf/services/configs/CommonServiceConfig.java

```diff
@@ -27,4 +27,13 @@
     + " created or rebalanced.  ceil(nBrokers * partitionsToBrokerRatio) is used to determine the actual number of "
     + "partitions when partitions are added or removed.";
 
+  public static final String TOPIC_REPLICATION_FACTOR_CONFIG = "topic-management.replicationFactor";
```

Comment:
Why is this config in `CommonServiceConfig` instead of `TopicManagementServiceConfig`?

Can you help correct typo in the PARTITIONS_TO_BROKER_RATO_CONFIG and PARTITIONS_TO_BROKER_RATIO_DOC? Also, that config is also used only in TopicManagementService. Can you move it as well?